service/src/main/java/uk/gov/hmcts/reform/fpl/utils/EmailNotificationHelper.java

```diff
@@ -0,0 +1,39 @@
+package uk.gov.hmcts.reform.fpl.utils;
+
+import uk.gov.hmcts.reform.fpl.model.CaseData;
+import uk.gov.hmcts.reform.fpl.model.Respondent;
+import uk.gov.hmcts.reform.fpl.model.RespondentParty;
+import uk.gov.hmcts.reform.fpl.model.common.Element;
+
+import java.util.Objects;
+
+import static org.apache.commons.lang3.ObjectUtils.defaultIfNull;
+import static org.apache.commons.lang3.StringUtils.isNoneBlank;
+import static org.springframework.util.CollectionUtils.isEmpty;
+
+public class EmailNotificationHelper {
+
+    private EmailNotificationHelper() {
```

Comment:
I seem to remember sonar cloud will complain about an empty constructor

Reply:
yes it did but the bug has been fixed [Sonar end](https://jira.sonarsource.com/browse/SONARJAVA-1138)